cpp/src/arrow/compute/exec/project_node.cc

```diff
@@ -91,6 +91,7 @@ class ProjectNode : public ExecNode {
 
   void InputReceived(ExecNode* input, ExecBatch batch) override {
     DCHECK_EQ(input, inputs_[0]);
+    ARROW_LOG(DEBUG) << "ProjectNode: >> project input >>" + batch.ToString();
```

Comment:
I take it this is debugging code specific to your test and this will be removed?

Reply:
sure